global/php/functions.php

```diff
@@ -21,6 +21,15 @@
 require_once('gitInfo.php');
 define("OSEKAI_VERSION", str_replace("\n", "", $gitHash)); // cache invalidation
 
+$server = $_SERVER['SERVER_SOFTWARE'];
+if(str_contains($server, "Apache")) {
+    define("ENVIRONMENT", "apache2");
+} else if(str_contains($server, "Development Server")) {
```

Comment:
```suggestion
} else if(str_contains($server, "PHP")) {
```
many web servers have Development Server in server software afaik , its a very general thing i think. it could really be anything at this point

Reply:
check now